fhir-server/src/main/java/com/ibm/fhir/server/resources/Capabilities.java

```diff
@@ -290,23 +291,23 @@ private CapabilityStatement buildCapabilityStatement() throws Exception {
         }
 
         // Build the lists of operations that are supported
-        List<OperationDefinition> systemOps = new ArrayList<>();
-        Map<ResourceType.Value, List<OperationDefinition>> typeOps = new HashMap<>();
+        Set<OperationDefinition> systemOps = new LinkedHashSet<>();
+        Map<ResourceType.Value, Set<OperationDefinition>> typeOps = new HashMap<>();
 
         FHIROperationRegistry opRegistry = FHIROperationRegistry.getInstance();
         List<String> operationNames = opRegistry.getOperationNames();
         for (String opName : operationNames) {
             FHIROperation operation = opRegistry.getOperation(opName);
             OperationDefinition opDef = operation.getDefinition();
-            if (opDef.getSystem().getValue()) {
+            if (opDef.getSystem().getValue().booleanValue()) {
```

Comment:
```suggestion
            if (Boolean.TRUE.equals(opDef.getSystem().getValue())) {
```

Reply:
Since `com.ibm.fhir.model.type.Boolean.value` is a `java.lang.Boolean` this suggestion will protect against the possibility of the value being `null`.